components/stargate/cmd/client.go

```diff
@@ -35,6 +35,10 @@ const (
 	StargateAuthClientIDFlag                     = "stargate-auth-client-id"
 	StargateAuthClientSecretFlag                 = "stargate-auth-client-secret"
 	StargateAuthIssuerURLFlag                    = "stargate-auth-issuer-url"
+	StargateAuthMaxRetriesFlag                   = "stargate-auth-max-retries"
```

Comment:
Why retry authentication?
I think we should let the underlying system (k8s in our context), retry if necessary.